sql/insert.go

```diff
@@ -88,6 +95,22 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 				b.Put(key, string(t))
```

Comment:
That's a good idea. Since the main disadvantage of this approach comes on the querying side, I'll let you make the decision in your select-on-secondary index work.